tslearn/matrix_profile.py

```diff
@@ -71,15 +71,21 @@ class MatrixProfile(TransformerMixin,
     Parameters
     ----------
     subsequence_length : int (default: 1)
-        Length of the subseries (also called window size) to be used for 
+        Length of the subseries (also called window size) to be used for
         subseries distance computations.
-    
+
+    algorithm : str (default: None)
+        Algorithm to use to compute the matrix profile.
+        Defaults to None to use the tslearn implementation.
+        Any other algorithm must be one of ['stump'], from the
+        stumpy python library.
```

Comment:
Is it possible to add a link to [stumpy docs](https://stumpy.readthedocs.io/en/latest) here (or to the [STUMPY API](https://stumpy.readthedocs.io/en/latest/api.html) page) so that users can read more about whether or not they want to use/install it?